### PR TITLE
Update FAQ to include common failures reasons with links

### PR DESCRIPTION
**Pull request by johnrsanders**
## Purpose

I updated the readme file FAQ to include the top four reasons this sample fails, to assist people in troubleshooting.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by kingsbayanalytics** [comment]
Thank you! Very helpful for a beginner. 4 was my issue

**Review by cam1911 — APPROVED** [review]

**Review by cam1911 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -124,3 +124,9 @@ Once in the web app:
 If you see this error while running `azd deploy`: `read /tmp/azd1992237260/backend_env/lib64: is a directory`, then delete the `./app/backend/backend_env folder` and re-run the `azd deploy` command.  This issue is being tracked here: https://github.com/Azure/azure-dev/issues/1237
 
 If the web app fails to deploy and you receive a '404 Not Found' message in your browser, run 'azd deploy'. 
+
+The most common reasons this sample fails are:
```
Comment (jongio) [inline README.md:128]: Combine this list with the items above.

File: README.md [hunk 2]
```diff
@@ -124,3 +124,9 @@ Once in the web app:
...
+1. The subscription (SubID) doesn't have access to the Azure OpenAI service. Please ensure the subID used in the access request process, is the one you're coding with aka.ms/oai/access
```
Comment (jongio) [inline README.md:129]:
```suggestion
1. The subscription (AZURE_SUBSCRIPTION_ID) doesn't have access to the Azure OpenAI service. Please ensure the AZURE_SUBSCRIPTION_ID used in the access request process, is the one you're coding with aka.ms/oai/access
```

File: README.md [hunk 3]
```diff
@@ -124,3 +124,9 @@ Once in the web app:
...
+2. You're attempting to create resources in regions not enabled for Azure OpenAI (e.g. East US 2, instead of East US), or where the model you're trying to use isn't enabled. See this matrix for what models are available in what regions aka.ms/oai/models
```
Comment (jongio) [inline README.md:130]: Link to the matrix?

File: README.md [hunk 4]
```diff
@@ -124,3 +124,9 @@ Once in the web app:
...
+3. You've exceeded a quota, most often number of resources per region. See this link for quotas and limits aka.ms/oai/quotas
```
Comment (jongio) [inline README.md:131]: Make a hyperlink.

**Review by jongio — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (johnrsanders) [reply README.md:128]: the items above seem to be random errors, not the top errors we see causing customers to fail here. Seems like two different lists to me, top issues. other issues.

File: README.md [hunk 5]
```diff
@@ -120,7 +120,10 @@ Once in the web app:
...
+3. You've exceeded a quota, most often number of resources per region. See this link for quotas and limits [https:/aka.ms/oai/quotas](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/quotas-limits)
+4. You've run the sample multiple times, and deleted the resources you've been creating each time, but are forgetting to purge them (Azure keeps resources for 48 hours unless you purge from soft delete). So you're getting same resource name not allowed conflicts. See this article: [https://learn.microsoft.com/en-us/azure/cognitive-services/manage-resources?tabs=azure-portal](https://learn.microsoft.com/en-us/azure/cognitive-services/manage-resources?tabs=azure-portal#purge-a-deleted-resource)
+1. If you see this error while running `azd deploy`: `read /tmp/azd1992237260/backend_env/lib64: is a directory`, then delete the `./app/backend/backend_env folder` and re-run the `azd deploy` command.  This issue is being tracked here: https://github.com/Azure/azure-dev/issues/1237
```
Comment (pablocastro) [inline README.md:128]: Nit: could you either use all "1." or all sequential numbers? Odd to have 1-4 and then 1, 1.

**Review by tonybaloney — APPROVED** [review]

**Comment by tonybaloney** [comment]
@jongio those changes are all done, please approve + merge

**Review by tonybaloney — APPROVED** [review]

**Review by jongio — APPROVED** [review]

**Pull request merged**